testing/features/components/asset_hyperlink.feature

```diff
@@ -7,6 +7,5 @@ Feature: Asset Hyperlink component
     Given an Asset Hyperlink component is on the page
     Then a link to the PDF is present
     And the label includes a name at the beginning
-    And the label includes an icon
```

Comment:
Removed this because it relied on some hyper-specific test code and it's a pretty visual test covered by backstop.